app/components/Settings/scheduler/login-modal.jsx

```diff
@@ -15,7 +15,10 @@ const PromptLoginModal = props => (
                 &times;
             </button>
             <Row className="header">
-                <p>To create a scheduled query, you'll need to be logged into Plotly.</p>
+                <p>
+                    To create a scheduled query, you'll need to be logged into Plotly. Logging in will reset your query,
+                    so please save it elsewhere before doing so.
```

Comment:
This is probably about as good as we can do I suppose. There's no way to squirrel the query away into local storage or a cookie or something and restore it after login?

Reply:
Happy to look into this on the next stage, but given time constraints, I don't feel comfortable implementing/testing this functionality before the release.

Reply:
OK. Any way we could make this warning very clear? put it on its own line, in bold? this is a pretty bad failure case if you've just iterated for an hour on some complex query.